subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,88 @@
+Plugins provide a means of reusing logic and configuration across multiple projects.
+With plugins, you can write a task once and use it in multiple builds. Or you can store common configuration,
+like logging, dependencies, and version management, in one place. This reduces duplication in
+build scripts.
 
-=== 3. Gradle has several fixed build phases
+=== Build Phases
 
-It's important to understand that Gradle evaluates and executes build scripts in three phases:
+Gradle evaluates and executes build scripts in three *build phases* of the <<build_lifecycle#build_lifecycle,Build Lifecycle>>:
 
- 1. Initialization
-+
+Initialization::
 Sets up the environment for the build and determine which projects will take part in it.
 
- 2. Configuration
-+
-Constructs and configures the task graph for the build and then determines which tasks need to run and in which order, based on the task the user wants to run.
+Configuration::
+Constructs and configures the task graph for the build. Determines which tasks need to run and in which order, based on the task the user wants to run.
 
- 3. Execution
-+
+Execution::
 Runs the tasks selected at the end of the configuration phase.
 
-These phases form Gradle's <<build_lifecycle#build_lifecycle,Build Lifecycle>>.
-
-[NOTE]
-.Comparison to Apache Maven terminology
-====
-Gradle's build phases are not like Maven's phases. Maven uses its phases to divide the build execution into multiple stages. They serve a similar role to Gradle's task graph, although less flexibly.
-
-Maven's concept of a build lifecycle is loosely similar to Gradle's <<base_plugin#sec:base_tasks,lifecycle tasks>>.
-====
-
-Well-designed build scripts consist mostly of <<authoring_maintainable_build_scripts#sec:avoid_imperative_logic_in_scripts,declarative configuration rather than imperative logic>>. That configuration is understandably evaluated during the configuration phase. Even so, many such builds also have task actions — for example via `doLast {}` and `doFirst {}` blocks — which are evaluated during the execution phase. This is important because code evaluated during the configuration phase won't see changes that happen during the execution phase.
-
-Another important aspect of the configuration phase is that everything involved in it is evaluated _every time the build runs_. That is why it's best practice to <<authoring_maintainable_build_scripts#sec:minimize_logic_executed_configuration_phase,avoid expensive work during the configuration phase>>. https://scans.gradle.com/[Build scans] can help you identify such hotspots, among other things.
-
-=== 4. Gradle is extensible in more ways than one
-
-It would be great if you could build your project using only the build logic bundled with Gradle, but that's rarely possible. Most builds have some special requirements that mean you need to add custom build logic.
```

Comment:
I really like these lines. Perhaps we can cut a lot of this content and put it in a section like 
customizing builds?

Reply:
Just added this information in the "design" section's Extensibility header.